src/bika/lims/browser/worksheet/views/printview.py

```diff
@@ -112,21 +110,12 @@ def getWSTemplates(self):
         """ Returns a DisplayList with the available templates found in
             templates/worksheets
         """
-        this_dir = os.path.dirname(os.path.abspath(__file__))
-        templates_dir = os.path.join(this_dir, self._TEMPLATES_DIR)
-        tempath = '%s/%s' % (templates_dir, '*.pt')
-        templates = [t.split('/')[-1] for t in glob.glob(tempath)]
         out = []
-        for template in templates:
-            out.append({'id': template, 'title': template[:-3]})
-        for templates_resource in iterDirectoriesOfType(self._TEMPLATES_ADDON_DIR):
-            prefix = templates_resource.__name__
-            templates = [tpl for tpl in templates_resource.listDirectory() if tpl.endswith('.pt')]
-            for template in templates:
-                out.append({
-                    'id': '{0}:{1}'.format(prefix, template),
-                    'title': '{0} ({1})'.format(template[:-3], prefix),
-                })
+        for template_id in self._TEMPLATES_LIST:
+            out.append({
+                "id": template_id,
+                "title": "{1} ({0})".format(*template_id.split(":")).replace(".pt", ""),
```

Comment:
Would you mind to split the ID on `_` and `capitalize` the parts?
Also I would appreciate if you could rename  `ar` -> `sample`;) I really don't like the current naming of these templates...

<img width="444" alt="Bildschirmfoto 2024-03-13 um 1 02 20 PM" src="https://github.com/senaite/senaite.core/assets/713193/6bc645b7-ef18-4483-b301-74e38f796f0c">

Reply:
So right? )

![Screenshot 2024-03-13 at 20 25 40](https://github.com/senaite/senaite.core/assets/15684023/e4374a9b-4d8a-45a4-a9fc-2c90324c1845)

Reply:
Awesome, much better! Thanks @Bugerman58